resources/assets/less/bem/btn-osu-big.less

```diff
@@ -51,8 +51,8 @@
   }
 
   &--account-edit {
-    display: block;
-    width: @avatar-size--base;
+    display: inline-block;
```

Comment:
the base block is already `display: inline-block`, so the entire line can be removed.